conf/kyuubi-defaults.conf.template

```diff
@@ -17,10 +17,19 @@
 
 ## Kyuubi Configurations
 
+# Details in https://kyuubi.readthedocs.io/en/master/deployment/settings.html
+
 #
-# kyuubi.authentication           NONE
-# kyuubi.frontend.bind.host       localhost
-# kyuubi.frontend.bind.port       10009
+# kyuubi.authentication                    NONE
+#
+# kyuubi.frontend.bind.host                <undefined>
```

Comment:
why use `<undefined>`?

Reply:
the default value is undefined, and Kyuubi will bind the first non-localhost network card.